### ISSUE TITLE
`<button>` elements in the editor cause parent forms to submit

### ISSUE THREAD

**Issue by plibither8** [josdejong/svelte-jsoneditor#11]
**Situation:** I have the json-editor as one of the children inside a `<form>` element. Clicking on the `<button>` elements of the json-editor, such as the "expand" button, causes the parent form to get submitted. This is because the `<button>` elements don't have `type="button"` attribute.

**Fix:** Add `type="button"` as an attribute to all `<button>`s used in the editor.

Should I open a PR for the same? Thanks :)

**Comment by plibither8** [comment]
Workaround, wrap it in it's own `form` element and intercept the submit events:

```svelte
<form on:submit|preventDefault|stopPropagation>
  <JSONEditor />
</form>
```

**Comment by josdejong** [comment]
Good point Mihir. I've fixed this via 64d873a. Ideally, the linter would protect against this. I gave it a short try but can't figure out how to get eslint+svelte3 configured to give an error on buttons without a `type` defined.

**Comment by plibither8** [comment]
Thanks Jos for the quick fix! I've not really configured ESLint with Svelte, so I won't be able to contribute much on that end :sweat_smile:.

**Comment by josdejong** [comment]
Fixed now in `v0.1.6`
